hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/api/filter/AccessLogFilter.java

```diff
@@ -114,6 +115,9 @@ public void filter(ContainerRequestContext requestContext,
                          executeTime, null, method, path, uri.getQuery());
             }
         }
+
+        // Unset the context in "HugeAuthenticator", need distinguish Graph/Auth server lifecycle
+        HugeGraphAuthProxy.resetContext();
```

Comment:
to make the code look more modular, can we add a method `HugeAuthenticator(Authorizer authorizer)`?  and here we can call `manager.unauthenticate(Authorizer authorizer)`, note authorizer=requestContext.getSecurityContext().

Reply:
> to make the code look more modular, can we add a method `HugeAuthenticator(Authorizer authorizer)`? and here we can call `manager.unauthenticate(Authorizer authorizer)`, note authorizer=requestContext.getSecurityContext().

thanks, but why we need transport the arg name authorizer?